modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -312,43 +314,28 @@ properties:
           enable_shared_bucket_access:
             type: boolean
             description: |
-              Introduced in Sync Gateway 1.5, this property specifies whether to enable Mobile-Web Data Sync and works in conjunction with the [import_docs](#databases-foo_db-import_docs) property.
+              Introduced in Sync Gateway 1.5, this property specifies whether to enable Mobile-Server Data Sync (a.k.a _mobile convergence_). You can learn more about this functionality [here](https://docs.couchbase.com/sync-gateway/current/shared-bucket-access.html)
```

Comment:
This is an MD style link
For asciidoc needs xref:sync-gateway::shared-bucket-access.adoc[here]

Reply:
I thought yaml was MD ... isn't that different

Reply:
Ah, my bad, I totally disregarded the file extension :face-palm:

Reply:
np